### PR TITLE
asadiqbal08/mit-xPRO-558 ZenDesk theme updates: round 1

### PR DESCRIPTION
**Pull request by asadiqbal08**
Task: [#558](https://github.com/mitodl/mitxpro/issues/558)

[Preview Theme HERE](https://xpro.zendesk.com/theming/theme/04173e1b-9cb8-463d-88d3-c72e8204abb6)

screencapture-xpro-zendesk-hc-en-us-2019-06-20-13_58_52

#### Acceptance Criteria:

- [x] "Contact US" => "Contact Us"
- [x] change all aqua text to the higher contrast / more accessible blue used elsewhere. See [InVision](https://projects.invisionapp.com/d/?origin=v7#/console/17338252/369007216/inspect) for details
- [x] "MITxPro" => "MIT xPRO"
- [x] links to “MITx PRO” should link to xpro.mit.edu not www.mit.edu
- [x] Can we change the top level of the breadcrumbs from "MIT xPRO" to "Help & FAQ"
- [x] hide or remove the "Sign Up" button 
- [x] remove "Search for answer" text 

@pdpinch for your information over it, I have added some custom settings in admin panel for that theme. A user can manage the fonts, color, header and footer (links as well) contents through it. Please take a look on screen below for an idea. 

- Example 1
Screen Shot 2019-06-18 at 4 56 54 PM

- Example 2
Screen Shot 2019-06-18 at 4 42 10 PM

### CODE REVIEW

**Comment by pdpinch** [comment]
Sorry, I thought I sent this to you on Sunday, but there's some font styling changes spec'ed here: https://projects.invisionapp.com/d/main#/console/17338252/369007216/preview

**Comment by asadiqbal08** [comment]
@pdpinch can you please specify what kind of font style need to be updated in that theme ? Is anything missed in the PR ? 

update: Had a one-2-one conversation with @abdulkdawson over it.

**Comment by asadiqbal08** [comment]
@pdpinch I have updated the theme and here is the latest image. Let me know are we good with that change here ? 
- Update the search area and button text

screencapture-xpro-zendesk-hc-en-us-2019-06-19-17_16_53

**Comment by pdpinch** [comment]
LGTM. There will be another round of changes once we've decided what to do with the rest of the feedback, but this is good for now. 

Can @tasawernawaz or @ahmed-arbisoft do a review?

File: style.css [hunk 1]
```diff
@@ -956,6 +956,50 @@ ul {
   padding: 10px 0 0;
 }
 
+.hero .search {
+  overflow: hidden;
+}
+
+.hero .search:before {
```
Comment (ahmed-arbisoft) [inline style.css:989]: Could move this into the block above with `&:before {`.

File: style.css [hunk 2]
```diff
@@ -956,6 +956,50 @@ ul {
...
+  right: 125px;
+}
+
+@media (min-width: 768px) {
```
Comment (ahmed-arbisoft) [inline style.css:993]: If this is only scoped to `.hero .search:before` we can move this into the block above.

File: style.css [hunk 3]
```diff
@@ -956,6 +956,50 @@ ul {
...
+  background: #355B6B;
+  font-size: 20px;
+  font-weight: 600;
+  height: 65px;
+  min-width: 100px;
+  padding: 0 10px;
+  text-transform: uppercase;
+}
+
+@media (min-width: 768px) {
```
Comment (ahmed-arbisoft) [inline style.css:1011]: Move into the block above.

File: style.css [hunk 4]
```diff
@@ -956,6 +956,50 @@ ul {
...
+  }
+}
+
+.hero .search input[type="search"] {
+  float: left;
+  width: calc(100% - 110px);
+  font-weight: 600;
+}
+
+@media (min-width: 768px) {
```
Comment (ahmed-arbisoft) [inline style.css:1023]: Move into block above.

File: templates/footer.hbs [hunk 5]
```diff
@@ -23,7 +23,7 @@
         </address>
       </div>
     <ul class="footer-nav">
-      <li><a href={{settings.mit_home}}>MITxPro Home</a></li>
+      <li><a href={{settings.mit_home}}>MIT xPro Home</a></li>
```
Comment (ahmed-arbisoft) [inline templates/footer.hbs]: Please correct me if I'm wrong, as per Peter's comments isn't it supposed to be `MIT xPRO` with capital PRO?

**Review by ahmed-arbisoft — APPROVED** [review]
Minor changes requested.

File: templates/footer.hbs (hunk 5 above)
Comment (asadiqbal08) [reply templates/footer.hbs]: yes.

File: style.css (hunk 1 above)
Comment (asadiqbal08) [reply style.css:989]: @ahmed-arbisoft it is compiled form of `scss`, what do want to do with `style.css` ?

**Comment by asadiqbal08** [comment]
@abdulkdawson Update the screenshot in description and added the `SUPPORT CENTER` text. take a look and let me know are we good with merge ?

**Comment by ahmed-arbisoft** [comment]
Nvm, go ahead for the merge, I didn't notice it was in the compiled CSS. Your SCSS looks fine. :+1:

**Review by pdpinch — APPROVED** [review]
Please go ahead and merge this and deploy it to ZenDesk. If there are any issues with the "Support Center" header, we can add them to the follow-up issue at https://github.com/mitodl/mitxpro/issues/594

**Pull request merged**